keps/sig-node/4817-resource-claim-device-status/README.md

```diff
@@ -438,6 +438,8 @@ network status.
 - Feature Gates are enabled by default.
 - No major outstanding bugs.
 - 1 example of real-world usage.
+    - The kubernetes-sigs/cni-dra-driver project as reference implementation for this feature.
+    - A new EndpointSlice Controller consuming the `ResourceClaim.Status.Devices[].NetworkData.Addresses` field.
```

Comment:
just it is important to mention that we need automated e2e test running periodically in beta, to take into consideration is not just to merge the code, is to make sure it is working and tested.
I'll be helping with that but I can not own it, I already maintain all the other testing ... so I expect the multi network community to own this CI , is that fair?

Reply:
Do you mean automated e2e tests running periodically for the cni-dra-driver and this new EndpointSlice Controller? I think that is fair yes.

Reply:
Speaking of testing: there's a test flake in the existing E2E (https://github.com/kubernetes/kubernetes/issues/130096), please fix before the beta graduation.